Assets/Scripts/Utilities/Utils.cs

```diff
@@ -115,6 +115,24 @@ public static Vector2 LatLngFromPosition(Vector3 pos, float rad)
         float lon = (float) Math.Atan(pos.x / pos.z) * -180 / Mathf.PI;
         return new Vector2(lat, lon);
     }
+
+    public static Color GetColorFromTexture(Renderer rend, Vector2 pixelUV)
+    {
+        if (rend == null)
+        {
+            Debug.Log("no renderer");
+            return Color.black;
+        }
+        Texture2D tex = rend.material.GetTexture("_MainTex") as Texture2D;
```

Comment:
Should we rename that texture to "_MainEarthText"?,  and/or maybe pass in the texture name to this function?

Reply:
Convention for the main texture of a shader is _MainTex - I believe I could have used material.mainTexture (as a shortcut) but I think because this is a custom shader that didn't work, despite the names being the same.